operations/addresses.md

```diff
@@ -211,48 +226,49 @@ Updates one or more existing addresses in the system, assigned to specified acco
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `ChainId` | string | optional | Unique identifier of the chain. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
-| `AddressUpdates` | array of [Account address updates](#account-address-update-parameters), max 1000 items | required | Collection of addresses to be updated. |
+| `AddressUpdates` | array of [Account address update parameters](addresses.md#account-address-update-parameters) | required, max 1000 items | Collection of addresses to be updated. |
 
 #### Account address update parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `AddressId` | string | required | Unique identifier of the address. |
-| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
-| `Line1` | [String update value](_objects.md#string-update-value) | optional | First line of the address. |
-| `Line2` | [String update value](_objects.md#string-update-value) | optional | Second line of the address. |
-| `City` | [String update value](_objects.md#string-update-value) | optional | The city. |
-| `PostalCode` | [String update value](_objects.md#string-update-value) | optional | Postal code. |
-| `CountryCode` | [String update value](_objects.md#string-update-value) | optional | ISO 3166-1 code of the [Country](countries.md#country). |
-| `CountrySubdivisionCode` | [String update value](_objects.md#string-update-value) | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Line1` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | First line of the address. |
+| `Line2` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | Second line of the address. |
+| `City` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | The city. |
+| `PostalCode` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | Postal code. |
+| `CountryCode` | [String update value](_objects.md#string-update-value) | optional, max length 8 characters | ISO 3166-1 code of the Country. |
+| `CountrySubdivisionCode` | [String update value](_objects.md#string-update-value) | optional, max length 8 characters | ISO 3166-2 code of the administrative division, e.g. DE-BW. |
+| ~~`AccountId`~~ | ~~string~~ | ~~required~~ | **Deprecated!** The value is ignored.|
```

Comment:
I don't understand why this value was not deleted, since the usage of it was removed 🤯 
I think we can just remove it.

Reply:
Yeah, or hide it with InternalImplementation for starters.